lib/membrane_h264_ffmpeg/parser/nalu.ex

```diff
@@ -31,11 +31,14 @@ defmodule Membrane.H264.FFmpeg.Parser.NALu do
               end)
               |> Map.new()
 
-  @spec parse(binary) :: {list, %{h264: any}}
-  def parse(access_unit) do
+  @type parse_option_t() :: {:discard_partial_nalu?, boolean()}
+  @type parse_options_t() :: list(parse_option_t())
```

Comment:
There is no sense in duplicating options in a list. Also, can we add documentation?

```suggestion
  @type parse_options_t() :: {:discard_partial_nalu?, boolean()}
```

Reply:
Options is usually a list. Otherwise, you need to pass an ugly tuple